tests/integration/test_auth.py

```diff
@@ -85,7 +85,10 @@ async def test_openfga_relation(self, ops_test: OpsTest):
                     )
                     result = await action.wait()
                     logger.info(f"attempt {i} -> action result {result.status} {result.results}")
-                    if result.status == "completed" and result.results == {"return-code": 0}:
+                    if result.status == "completed" and result.results == {
```

Comment:
nitpick: This is ugly. A more elegant way does not come to my mind at the moment but you can at least put the comparisons into variables and use them in the branch check.